lib/src/facts/windows/collection.cc

```diff
@@ -82,7 +82,7 @@ namespace facter { namespace facts {
             add(make_shared<windows::operating_system_resolver>(shared_wmi));
             add(make_shared<windows::processor_resolver>(shared_wmi));
             add(make_shared<windows::virtualization_resolver>(shared_wmi));
-            add(make_shared<windows::uptime_resolver>(shared_wmi));
+            add(make_shared<windows::uptime_resolver>());
```

Comment:
Let's move this up to line 77 so it's still resolved if WMI is unavailable.

Reply:
Good pt on the above - will do.